packages/api-explorer/src/JsonForm/antd-theme-json-editor.js

```diff
@@ -47,6 +47,17 @@ const antdTheme = (JSONEditor) => class extends JSONEditor.defaults.themes.boots
     el.style.display = 'inline-flex'
     return el
   }
+
+  addInputError (input, text) {
```

Comment:
This could be made inside json-editor with an option

Reply:
In this way you do not need to `return input` (that is made only in bootstrap4's method, but the interface should be equals in all themes)